src/chartserver/handler_repo.go

```diff
@@ -55,7 +55,7 @@ func (c *Controller) getIndexYaml(namespaces []string) (*helm_repo.IndexFile, er
 	// Receive error
 	errorChan := make(chan error, 1)
 	// Signal chan for merging work
-	mergeDone := make(chan struct{}, 1)
+	mergeDone := make(chan struct{})
```

Comment:
hi @Iceber thanks for you contribuiton, because chartmuseum is planned to be removed from v2.8 and we will not maintain the code on chartmuseum except for cve.

Reply:
Okay, I'll delete the changes in chartserver and just modify *cmd/migrate-patch* and *core*